docs/graphene-workload-tutorial/workload/graphene_sgx/build_gsc_workload.sh.template

```diff
@@ -38,29 +38,13 @@ if [ "$GSC_IMAGE_EXISTS" = "yes" ]; then
 fi
 
 # Manifest files
-MANIFEST_FILE_DIR="${TCF_HOME}/tc/graphene/python_worker/graphene_sgx/manifest"
-MANIFEST_FILES="python.manifest
-sh.manifest
-gcc.manifest
-collect2.manifest
-ld.manifest"
-# Generate list of manifest files
-LIST_MANIFEST_FILES=""
-for f in $MANIFEST_FILES
-do
-  FILE_NAME=${MANIFEST_FILE_DIR}/$f
-  if [ ! -f $FILE_NAME ]; then
-    echo "ERROR:Manifest file $FILE_NAME doesn't exist"
-    exit
-  fi
-  LIST_MANIFEST_FILES+=${MANIFEST_FILE_DIR}/$f
-  LIST_MANIFEST_FILES+=" "
-done
-echo $LIST_MANIFEST_FILES
+
+MANIFEST_FILE="${TCF_HOME}tc/graphene/python_worker/graphene_sgx/manifest/avalon.manifest"
+echo $MANIFEST_FILE
 
 # Build image
 echo "Build unsigned GSC image"
-./gsc build --insecure-args $IMAGE_NAME $LIST_MANIFEST_FILES
+sudo ./gsc build --insecure-args $IMAGE_NAME $MANIFEST_FILE
```

Comment:
Why sudo needed here? gsc binary should run with user permission itself.

Reply:
yes, sudo is not required. gsc runs with user permission, tested with gramine production release.

Removed sudo, in all the gsc build scripts